consensus/src/main/java/org/ethereum/beacon/consensus/transition/InitialStateTransition.java

```diff
@@ -67,9 +69,11 @@ public BeaconState apply(BeaconBlock block, BeaconState state) {
 
     // Randomness and committees
     builder
-        .withLatestRandaoMixes(nCopies(BeaconState.LATEST_RANDAO_MIXES_LENGTH, Hash32.ZERO))
+        .withLatestRandaoMixes(
+            nCopies(chainSpec.getLatestRandaoMixesLength().getIntValue(), Hash32.ZERO))
         .withLatestVdfOutputs(
-            nCopies(BeaconState.LATEST_RANDAO_MIXES_LENGTH / Epoch.LENGTH, Hash32.ZERO))
+            nCopies(
+                chainSpec.getLatestRandaoMixesLength().getIntValue() / Epoch.LENGTH, Hash32.ZERO))
```

Comment:
I suppose `Epoch.LENGTH` should also be in the `ChainSpec` ?

Reply:
Yep, it's just in the next section of constants. Not yet covered